tests/expectations/core/test_expect_column_values_to_be_decreasing.py

```diff
@@ -41,5 +41,5 @@ def test_expect_column_values_to_be_decreasing_warn_parse_strings_as_datetimes(
             column="a", parse_strings_as_datetimes=True
         )
     assert 'The parameter "parse_strings_as_datetimes" is deprecated' in str(
-        record.list[0].message
+        record.list[1].message
```

Comment:
Adjusting of warning is needed now that we output one for Expectation and Validator-level `result_format` configurations